labs/github/github.py

```diff
@@ -0,0 +1,79 @@
+import requests
+from labs.config import GITHUB_ACCESS_TOKEN
+from labs.config import GITHUB_API_BASE_URL
+from labs.config import GITHUB_OWNER
+from labs.config import GITHUB_REPO
+from labs.config import GITHUB_USERNAME
+from labs.config import get_logger
+
+class Github:
+    def __init__(self, access_token=GITHUB_ACCESS_TOKEN, api_base_url=GITHUB_API_BASE_URL, 
+                 owner=GITHUB_OWNER, repo=GITHUB_REPO, username=GITHUB_USERNAME):
+        self.access_token = access_token
+        self.api_base_url = api_base_url
+        self.owner = owner
+        self.repo = repo
+        self.username = username
+        self.logger = get_logger(__name__)
+
+    def get_issues_from_repo(self, assignee=None, state='open', per_page=100):
```

Comment:
this could be called just get_issues or issues, since we're not getting issue from anyplace else than a repo in this class